project/overview/setup.rst

```diff
@@ -0,0 +1,63 @@
+========================
+Get Started with Project
+========================
+
+Managing projects means managing people, tasks, due dates, budgets and predicted return on
+investments. For all of it, you need a solution that allows you to reduce risks, get information at
+a glance, have an overview of your progresses and stay organized. And, from start to finish, to be
+able to run multiple projects in parallel easily.
+
```

Comment:
Since this intro states what are the users' needs (this approach feels more like advertising, but why not :) ), then it would be great to have a small sentence to say that Odoo Projects meets all these requirements + "here is how to start a project".

Reply:
I added a sentence where I mention Odoo, but the 'here is how..' I don't believe is necessary. It is a sentence for eLearning but not for doc (: